libc-test/build.rs

```diff
@@ -302,6 +302,7 @@ fn main() {
             // Fixup a few types on windows that don't actually exist.
             "time64_t" if windows => "__time64_t".to_string(),
             "ssize_t" if windows => "SSIZE_T".to_string(),
+            "pid_t" if windows => "int".to_string(),
```

Comment:
Hm this is completely renaming the type? Does Windows not have a `pid_t` type in MinGW or MSVC?

Reply:
Correct.

Reply:
`getpid` aka `_getpid` just returns 'int'.

Reply:
Ah ok, in that case we tend to prefer to not try to abstract over the underlying platform too much in the sense that we don't want to manufacture types in Rust-lang which don't actually exist on the underlying system.

Reply:
Grumble. This is a common thing to have in cross platform code. Windows is the only platform without pid_t. Leaving it out of libc forces every lib and program to have ifdefs in their own code.

Reply:
I also learned how to fix the compilation error by copying existing Windows work arounds so this isn't the first one to be added.

Reply:
Oh sure yeah it makes sense that this is a pain, but it's not `libc`'s job to sovle this, that's up to another crate.